src/components/dapp-staking/my-staking/DynamicAdsArea.vue

```diff
@@ -90,7 +90,7 @@ export default defineComponent({
         } else {
           return require('/src/assets/img/ic_astar_farm.svg');
         }
-      } else if (index === 2) {
+      } else if (index === 1) {
```

Comment:
This is overriding code for the first two campaigns.  Their "img" can remain empty.